docs/docs/platform/quickstarts/nextjs.mdx

```diff
@@ -0,0 +1,744 @@
+---
+title: 'Quickstart: Next.js'
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Quickstart: Next.js
+
+## Introduction
+
+This quickstart guide provides the steps you need to build a simple Next.js app
+powered by Nhost for the backend. It includes:
+
+- Database: [PostgreSQL](https://www.postgresql.org/)
+- Instant GraphQL API: [Hasura](https://hasura.io/)
+- Authentication: [Hasura Auth](https://github.com/nhost/hasura-auth/)
+- Storage: [Hasura Storage](https://hub.docker.com/r/nhost/hasura-storage)
+
+By the end of this guide, you'll have a full-stack app that allows users to log
+in to access a protected dashboard and update their profile information.
+
+## Prerequisites
+
+Before getting started, let's make sure that your development environment is
+ready.
+
+You'll need **Node.js** version 10.13 or later: [install it from here](https://nodejs.org/en/).
+
+## Project setup
+
+### Create a new Nhost app
+
+First things first, we need to create a new Nhost project.
+
+So, log in to your Nhost dashboard and click the **Create your first app**
+button.
+
+![nhost-first-app](/img/quickstarts/create-app-step-1.png)
+
+Next, give your new Nhost app a name, select a geographic region for your Nhost
+services and click **Create App**.
+
+![nhost-first-app](/img/quickstarts/create-app-step-2.png)
+
+After a few seconds, you should get a PostgreSQL database, a GraphQL API with
+Hasura, file storage, and authentication set up.
+
+:::info
+You can also connect your Nhost app to a GitHub repository. When you do this, any updates you push to your code will automatically be deployed. [Learn more](https://docs.nhost.io/platform/github-integration).
+:::
+
+## Initialize the app
+
+### Create a Next.js app
+
+The simplest way to create a new Next.js application is by using the tool called
+`create-next-app`, which bootstraps a Next.js app for you without the hassle of
+configuring everything yourself.
+
+So, open your terminal, and run the following command:
+
+```bash
+npx create-next-app my-nhost-app --example "https://github.com/nhost/quickstart-nextjs"
+```
+
+:::info
+This command uses an [existing template](https://github.com/nhost/quickstart-nextjs), through the `--example` flag, which already contains the React components and pages we'll use for this guide.
+:::
+
+You can now `cd` into your project directory:
+
+```bash
+cd my-nhost-app
+```
+
+And run the development server with the following command:
+
+<Tabs>
```

Comment:
I would change to `<Tabs groupId="package-manager">` to [sync tab choices](https://docusaurus.io/docs/next/markdown-features/tabs#syncing-tab-choices)

Reply:
Much better. Done.